src/actions/config.js

```diff
@@ -68,6 +68,22 @@ function parseConfig(data) {
   return config;
 }
 
+async function getConfig(file, isPreloaded) {
+  const response = await fetch(file, { credentials: 'same-origin' });
+  if (response.status !== 200) {
+    if (isPreloaded) return parseConfig('');
+    throw new Error(`Failed to load config.yml (${ response.status })`);
+  }
+  const contentType = response.headers.get('Content-Type') || 'Not-Found';
+  const isYaml = contentType.indexOf('yaml') !== -1;
```

Comment:
There are different type strings for yaml that can exist, so this should allow for each.